etna/transforms/encoders/categorical.py

```diff
@@ -0,0 +1,235 @@
+import warnings
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+from sklearn import preprocessing
+from sklearn.utils._encode import _check_unknown
+from sklearn.utils._encode import _encode
+from sklearn.utils._encode import _unique
+
+from etna.transforms.base import PerSegmentWrapper
+from etna.transforms.base import Transform
+
+
+class LE(preprocessing.LabelEncoder):
+    def transform(self, y: pd.Series, strategy):
```

Comment:
It is better to create Enum class like in other cases like [this](https://github.com/tinkoff-ai/etna/blob/master/etna/transforms/missing_values/imputation.py)